layouts/partials/footer.html

```diff
@@ -2,7 +2,7 @@
     <div class="container px-10 flex py-5 gap-3">
         <div class="text-white text-xs sm:text-base">
             <p>
-                <strong>{{.Site.Params.Copyright }}</strong>
+                <strong>Copyright &copy; {{now.Format "2006"}} Buffalo. {{.Site.Params.Copyright }}</strong>
```

Comment:
could it be `2016-{{ now.Format "2006" }}` to the result is shown as `2016-2023`? I think the copyright statement should include the published date or the range of publishment.

Reply:
> 
Yes, that makes sense, however, the copyright statement is taken from the yml file and I just leave the rest of the copyright text in the footer, I concatenate the current year and nothing else would have to be changed because of the existing languages. Does it make sense?

Reply:
@paganotoni Got it. Thanks!

Reply:
Yes, that is what I meant. It depends on what we published, we are publishing a set of content from 2016 to now.
Thanks for your comment @paganotoni!